src/content/docs/en/guides/images.mdx

```diff
@@ -457,13 +200,34 @@ For remote images, use the image's **full URL** as the `src` value:
 
 ### Choosing `<Image />` vs `<img>`
 
-The `<Image />` component optimizes your image and infers width and height (of local images) based on the original aspect ratio to avoid CLS.
+The `<Image />` component optimizes your image and infers width and height (for images it can process) based on the original aspect ratio to avoid CLS. It is the preferred way to use images in `.astro` files whenever possible.
 
 Use the HTML `<img>` element when you cannot use the `<Image />` component, for example:
   - for unsupported image formats
   - when you do not want your image optimized by Astro
   - to access and change the `src` attribute dynamically client-side
 
+### Setting Default Values
+
+Currently, there is no way to specify default values for all `<Image />`  or `<Picture/>` components. Required attributes should be set on each individual component.
+
+As an alternative, you can wrap these components in another Astro component for reuse. For example, you could create a component for your blog post images that receives attributes as props and applies consistent styles to each image:
+
+```astro title="src/components/BlogPostImage.astro"
+---
+import { Image } from 'astro:assets';
+
+const { src, ...attrs } = Astro.props;
+---
+<Image src={src} {...attrs} />
+
+<style>
+  img :global(img), svg {
```

Comment:
It seems weird to me to target an image in an image... Doesn't the `<Image />` component return an `<img />` tag in all cases? (whether the source is a jpg or an svg) So I would say we just need `img` here... but I am not sure (maybe `:global` is required).
```suggestion
  img {
```

Reply:
I will need to call in @delucis for assistance here!

Reply:
I've never used `<Image />` with an svg so I might be wrong... What seemed strange to me was mainly `img :global(img)`.

Reply:
+1 to Armand. SVGs are not processed in `<Image>` besides retaining `width` & `height` attributes. `:global` is related with [css modules](https://github.com/css-modules/css-modules/blob/master/docs/composition.md#exceptions) and is throwing me off too

Reply:
Yeah the styles here don’t really make sense. You can’t have `<img><img></img>` and this style would create a selector like `img[data-astro-cid-XXX] img` so wouldn’t actually ever match what is generated by this component. The `svg` is also not relevant because even when using `<Image>` with an SVG file, you still get an `<img>` tag, not an inline `<svg>`.

I think @ArmandPhilippot’s suggestion is correct. I did a quick test and it appears that `<Image>` inherits the containing component’s scoped classes, so just `img {}` will work well in this case — no need for `:global()` (which would also introduce other issues, as it would impact all images on the site).

Reply:
Thank you all!